integrations/cortex/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Cortex",
+  "version": "1.0.0",
+  "overview": "View LaunchDarkly feature flags directly in the catalog and alert on Scorecards in Cortex.",
+  "description": "With the integration, you can view and monitor feature flags in the Cortex catalog. You can also write Scorecard rules to enforce in Cortex feature flag usage and drive operational maturity.",
+  "author": "Cortex",
+  "supportEmail": "support@cortex.io",
+  "links": {
+    "site": "https://www.cortex.io/home",
+    "launchdarklyDocs": "https://docs.launchdarkly.com/integrations/cortex",
+    "supportWebsite": "https://support.getcortexapp.com/",
+    "privacyPolicy": "https://www.cortex.io/legal/privacy-policy"
+  },
+  "categories": ["developer-tools", "infrastructure"],
```

Comment:
@InTheCloudDan Thoughts on adding an "internal-developer-portal" category?

Reply:
@cristinabuenahora I'll chat with our Docs team about it. There is additional work we're going to be doing to align our tags in-app with better categorization but I don't want to block this PR on it. If we do add it I'll come back and update!